src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -78,6 +81,12 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
     this.callbackErrorsSubject.subscribe(errorEvent => {
       this.error = errorEvent;
     });
+    this.errorSubscription = this.errorNotifierService.error.subscribe(error => {
+      if (error.status && error.status !== 401 && error.status !== 403) {
+        this.error = error;
```

Comment:
can we have a situation where `error.status` is empty or null and so even though there is an `error` the if it's not executed hence the error not displayed?